test/util/http/data.js

```diff
@@ -84,12 +83,17 @@ const responsesByComponent = {
     ]
   }),
   FormPreview: componentResponses({
-    form: () => testData.extendedForms.last(),
-    xml: () => mockResponse.of(simpleXml)
+    form: [
+      ({ url }) => /^\/v1\/projects\/\d+\/forms\/[^/]+$/.test(url),
```

Comment:
I'm trying to figure out a way to make this part easier. Suggestions welcome! Maybe we could use a library that converts route paths to regular expressions. 🤔

Reply:
what do you think about this function:

```
const DIGITS = 'DIGITS';
const ALPHA_NUMERIC = 'ALPHA_NUMERIC';

const apiPathToRegex = (apiPathWithPlaceholders) => {
  const DIGITS_REGEX = '\\d+';
  const ALPH_NUMERIC_REGEX = '[^/]+';

  return new RegExp(apiPathWithPlaceholders
    .replace(DIGITS, DIGITS_REGEX)
    .replace(ALPHA_NUMERIC, ALPH_NUMERIC_REGEX));
};
```

and then

```
apiPathToRegex(apiPaths.form(DIGITS, ALPHA_NUMERIC)).test(url)
```